### PR TITLE
Nested stores event bubbling

### PR DESCRIPTION
**Pull request by samuelsimoes**
#21

### CODE REVIEW

File: src/fluxo.object_store.js [hunk 1]
```diff
@@ -117,6 +151,10 @@ export default class {
   unsetAttribute  (attribute, options) {
     options = options || {};
 
+    if (this.data[attribute] instanceof this.constructor) {
+      this.stopListenStoreAttribute(attribute);
+    }
+
```
Comment (sobrinho) [inline src/fluxo.object_store.js:180]: `this.constructor` will be always `Fluxo.ObjectStore`?

If I have a store A inheriting from `Fluxo.ObjectStore` and then a store B inheriting from A, the constructor of B will be `Fluxo.ObjectStore` or `A`?

Comment (samuelsimoes) [inline src/fluxo.object_store.js:180]: It's wrong... Must be a comparison with `Fluxo.ObjectStore`, I will fix.

**Comment by samuelsimoes** [comment]
@sobrinho fixed :point_up:

**Comment by sobrinho** [comment]
@samuelsimoes the same applies to the others `this.constructor`, no?

**Comment by samuelsimoes** [comment]
If I understood your question the B instance will respond true for ObjectStore, A and B, like this:
fluxo

**Pull request merged**